### ISSUE TITLE
Field metadata: add plot spacing information to field design

### ISSUE THREAD

**Issue by bauchetg** [solgenomics/sgn#2296]
Expected Behavior 
--------------------------------------------------------------------------
We currently have plot length and width cvterms
Requested by IITA, the plot spacing terms: "inter-row space" and "inter-plot space", ex:
Plot size: 5.6m x 4m
Spacing :   1m x 0.8m

For Bugs:
---------

**Comment by bellerbrock** [comment]
@bauchetg Makes sense to have plot spacing information. Does IITA vary their spacing much in different trial types?

NCSU have expressed interest in specifying the # plants per plot in the trial design, rather than the plot dimensions. They use consistent spacing, always 1 foot between plants in the row, and 3 feet between rows. The spacing and the number of plants determines the plot size and field size. And when it's time for analysis the number of plants is important for expressing plot yield on a per plant basis so that comparisons across trials with different plot sizes are meaningful.

**Comment by bauchetg** [comment]
Hello Bryan, yes we are currently implementing this with Nick and Lukas under this [branch](https://github.com/solgenomics/sgn/tree/topic/add_plantnumber_to_trial) Plant per plot is part of the variable as spacing information. So far info stored use metric system, do you think we shall have non metric too?

**Comment by bellerbrock** [comment]
Any updates on this? Also yes, it would require larger changes but it could make sense to let users edit and track theses properties by trial stage. Maybe store them as breeding program props, then during trial design fill in the defaults according to the user's program and what stage of trial they have selected.

**Comment by bauchetg** [comment]
to be added:
Spacing_Length
Spacing_Width
Plant Per Plot
Number of Harvested Plants Per   Plot

Allow more flexible plot width/length entry from UI

**Comment by lukasmueller** [comment]
Should be done at the same time as #4427

**Comment by chris263** [comment]
Hello @lukasmueller, Should we get back to this discussion or close this ticket?
